src/praisonai-code/praisonai_code/cli/state/project_sessions.py

```
@@ -9,7 +9,39 @@
+        import shutil
+
+        new_dir.mkdir(parents=True, exist_ok=True)
+        for entry in legacy_dir.iterdir():
+            if not entry.is_file():
+                continue
+            target = new_dir / entry.name
+            if target.exists():
+                continue
+            shutil.copy2(entry, target)
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)